server/pom.xml

```diff
@@ -46,7 +46,7 @@
         <dependency>
             <groupId>com.github.StuPro-TOSCAna</groupId>
             <artifactId>cloudformation-builder</artifactId>
-            <version>72c1e0c2e80808d69b4aeb185a7533ebfa2ac3da</version>
+            <version>5b856331b4d8718e83bc715be4a827adbc1db61c</version>
```

Comment:
You could try this with the newest hash 1294a9af6c8dff82d7cca757ecbbdb2a004643c1